lib/client-assets.php

```diff
@@ -799,6 +799,41 @@ function gutenberg_capture_code_editor_settings( $settings ) {
 	$gutenberg_captured_code_editor_settings = $settings;
 	return false;
 }
+/**
+ * Retrieve a stored autosave that is newer than the post save.
+ *
+ * Deletes autosaves that are older than the post save.
+ *
+ * @param  WP_Post $post Post object.
+ * @return WP_Post|boolean The post autosave. False if none found.
+ */
+function get_autosave_newer_than_post_save( $post ) {
+	// Add autosave data if it is newer and changed.
```

Comment:
If we need to do that in PHP, it means the API is not working well. Why do we need to two "get" endpoints there for autosaves? They are really pointless if we can't get the latest revision from the API.

Reply:
Yea, we could do this with the API instead, good suggestion.

Reply:
👍